src/collections/Events.ts

```diff
@@ -1,12 +1,68 @@
-import type { CollectionConfig } from 'payload'
+import { isAdmin } from '@/access/isAdmin'
+import { isOrganizer } from '@/access/isOrganizer'
+import type { CollectionConfig, PayloadRequest } from 'payload'
 
 export const Events: CollectionConfig = {
   slug: 'events',
+  admin: {
+    useAsTitle: 'title',
+  },
+  versions: {
+    drafts: true,
+  },
+  access: {
+    read: ({ req }: { req: PayloadRequest }) => {
+      return isOrganizer({ req })
+    },
+    create: ({ req }: { req: PayloadRequest }) => {
+      return isOrganizer({ req })
+    },
+    delete: ({ req }: { req: PayloadRequest }) => {
+      return isAdmin({ req })
+    },
+    update: ({ req }: { req: PayloadRequest }) => {
+      if (isOrganizer({ req })) return true
+      return {
+        or: [
```

Comment:
I don't think you need this condition for update. Organizer's have all permissions - which is unlike the editor/contributor relationship.